html/seed-display.html

```diff
@@ -134,84 +147,103 @@ <h2 class="mt-6 text-center text-3xl font-extrabold text-gray-900">
 
     <script>
-      function copySeed() {
-        const { value: text } = document.getElementById("signup-passphrase-text")
-
-        navigator.clipboard.writeText(text).then(() => {
-          const button = document.getElementById("copy-seed-button");
-          const copied = document.getElementById("copy-seed-copied");
-          
-          button.classList.add("hidden")
-          copied.classList.remove("hidden")
-
-          window.setTimeout(() => {
-            button.classList.remove("hidden")
-            copied.classList.add("hidden")
-          }, 3000)
-        }, (error) => {
-          console.error('Async: Could not copy text: ', error);
-        });
-      }
-
-      function outFunc() {
-        var tooltip = document.getElementById("copy-tooltip");
-        tooltip.innerHTML = "Copy to clipboard";
-      }
-
-      function showPassword() {
-        const seedInput = document.getElementById("signin-passphrase-text");
-        const triggerText = document.getElementById("show-seed-text");
-        const triggerIconOn = document.getElementById("show-seed-on");
-        const triggerIconOff = document.getElementById("show-seed-off");
-
-        if (seedInput.type === "password") {
-          seedInput.type = "text";
-          triggerText.textContent = "Hide passphrase"
-          triggerIconOn.classList.add("hidden");
-          triggerIconOff.classList.remove("hidden");
-        } else {
-          seedInput.type = "password";
-          triggerText.textContent = "Show passphrase"
-          triggerIconOn.classList.remove("hidden");
-          triggerIconOff.classList.add("hidden");
-        }
-      }
-
-      function confirmStoredPassphrase() {
-        const seedConfirm = document.getElementById("seed-confirm");
-        const signupSubmit = document.getElementById("signup-submit");
-
-        if(seedConfirm.checked) {
-          signupSubmit.removeAttribute("disabled")
-        } else {
-          signupSubmit.setAttribute("disabled", "disabled")
-        }
-      }
+     function copySeed() {
+         const { value: text } = document.getElementById("signup-passphrase-text")
+
+         navigator.clipboard.writeText(text).then(() => {
+             const button = document.getElementById("copy-seed-button");
+             const copied = document.getElementById("copy-seed-copied");
+
+             button.classList.add("hidden")
+             copied.classList.remove("hidden")
+
+             window.setTimeout(() => {
+                 button.classList.remove("hidden")
+                 copied.classList.add("hidden")
+             }, 3000)
+         }, (error) => {
+             console.error('Async: Could not copy text: ', error);
+         });
+     }
+
+     function outFunc() {
+         var tooltip = document.getElementById("copy-tooltip");
+         tooltip.innerHTML = "Copy to clipboard";
+     }
+
+     function showPassword() {
+         const seedInput = document.getElementById("signin-passphrase-text");
```

Comment:
`nit` possible to define these elements outside of the function to avoid getting them from the DOM every time